bundles/GraphicServiceOrder/Resources/views/createOrderSubmitted.html.twig

```diff
@@ -10,16 +10,19 @@
     <p class="card-text">{{ 'service_order_form.receipt.sub_text'|trans }}<br>
   </div>
 
-  {#
   <h3 class="mt-4">{{ 'service_order_form.job_description.header'|trans }}</h3>
+  <div class="mb-2">
+    <span class="label"><strong>ID:</strong></span>
+    <span class="value">{{ order.issueKey }}</span>
+  </div>
   <div class="mb-2">
     <span class="label"><strong>{{ 'service_order_form.job_description.title.label'|trans }}:</strong></span>
-    <span class="value">{{ form_data.form.job_title }}</span>
+    <span class="value">{{ order.jobTitle }}</span>
   </div>
   <div class="mb-2">
     <span class="label"><strong>{{ 'service_order_form.job_description.orders.label'|trans }}:</strong></span>
     <ul>
-      {% for line in form_data.form.order_lines %}
+      {% for line in order.orderLines %}
```

Comment:
Do we know that `order.orderLines` is not empty? Otherwise, wrap `<ul/>` with `{% if … %}…{% endif %}` (cf. comment below).

Reply:
fixed